### PR TITLE
site: improve scrollbar style of site

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 优化滚动条样式 |
| 🇨🇳 Chinese | 优化滚动条样式 |

### CODE REVIEW

File: .dumi/theme/slots/Content/DocAnchor.tsx [hunk 1]
```diff
@@ -10,7 +10,8 @@ export const useStyle = createStyles(({ token, css }) => {
   return {
     anchorToc: css`
       scrollbar-width: thin;
-      scrollbar-color: unset;
+      scrollbar-color: #eaeaea transparent;
+      scrollbar-gutter: stable;
```
Comment (afc163) [inline .dumi/theme/slots/Content/DocAnchor.tsx:14]: 这个全局改一次不就行了么？

Comment (li-jia-nan) [reply .dumi/theme/slots/Content/DocAnchor.tsx:14]: 目前没有相关的 token

Comment (afc163) [reply .dumi/theme/slots/Content/DocAnchor.tsx:14]: 我看你在 html 里改了。

Comment (li-jia-nan) [reply .dumi/theme/slots/Content/DocAnchor.tsx:14]: 我试试能不能用 inherit 继承 html 的

Comment (afc163) [reply .dumi/theme/slots/Content/DocAnchor.tsx:14]: inherit 也不需要吧

Comment (li-jia-nan) [reply .dumi/theme/slots/Content/DocAnchor.tsx:14]: 试了一下，color 可以继承 html，但是 width 和 gutter 不行，我改一下

**Review by afc163 — APPROVED** [review]

**Pull request merged**